recipes/crane/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set name = "crane" %}
+{% set version = "0.19.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/go-containerregistry/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 2a03430b0c9a8cf3fdfde2608aed99f851e5c57dfed8246bcc637d03a7ab0048
+
+build:
+  number: 0
+  script:
+    - go build -v -ldflags "-w -s -X github.com/google/go-containerregistry/cmd/crane/cmd.Version={{ version }}" -o $PREFIX/bin/crane ./cmd/crane  # [not win]
+    - go install -v -ldflags "-w -s -X github.com/google/go-containerregistry/cmd/crane/cmd.Version={{ version }}" ./cmd/crane  # [win]
```

Comment:
For `go` and `rust` recipes, it would be best to add the `go-licenses` tooling to dump all the licenses used in a folder, and include it in the package.

You can find an example e.g. in the [`steampipe-feedstock`](https://github.com/conda-forge/steampipe-feedstock/blob/2768a9b5137f03a4aa41053ccb90c7ecc040cff0/recipe/meta.yaml#L35)

go-licenses save . --save_path=./license-files || true